.gitignore

```diff
@@ -1,13 +1,13 @@
-/target
-/local
+/**/target
```

Comment:
-1 because
1) all .gitignore should be as consistent as possible
2) it could be possible we have a subpackage target somewhere (think hypothetically org.optaplanner.core.builder.target.TargetDirectoryCreator), not just as a java package, but as a resource or dataset (think data/roadef2018/darts/target/target1dartsA.xml), so we don't do recursive ignores, but have the same .gitnore file in all dirs with a pom.xml file.